### PR TITLE
Fix #16033: Refactor remaining raw constants into sprites.h #16033

### LINKED ISSUE #16033: Refactor remaining raw constants into sprites.h
Constants like ```29046``` as seen here:
https://github.com/OpenRCT2/OpenRCT2/blob/eba42a958465e7d882fa1170ad8332daa4e2bf0c/src/openrct2/ride/VehiclePaint.cpp#L3049

Should be named ```SPR_SOME_NAME``` and put into sprites.h, there might be arrays that hold those. It should be easy to find them by looking for calls to PaintAddImageAsChild or PaintAddImageAsParent.

### PR DESCRIPTION
**Pull request by sammy-soto10**
Renamed the constants from the VehiclePaint.cpp and put the sprites in the sprite.h file. If there are any constants I missed or if you would like me to rename the sprites I would be happy to do it. 

Also sorry for the commit titles I accidentally forgot the rest of the message when making it and I still haven't learned how to change commit messages.

### CODE REVIEW

File: src/openrct2/sprites.h [hunk 1]
```diff
@@ -1121,4 +1121,10 @@ enum
     SPR_IMAGE_LIST_LENGTH = 1000000,
     SPR_IMAGE_LIST_BEGIN = SPR_SCROLLING_TEXT_END,
     SPR_IMAGE_LIST_END = SPR_IMAGE_LIST_BEGIN + SPR_IMAGE_LIST_LENGTH,
+
+    SPR_PAINT_SPLASH_EFFECT_1 = 29014,
```
Comment (duncanspumpkin) [inline src/openrct2/sprites.h:1125]: We want to ideally list all of the missing elements i.e. from 29014 -> 29045 are all for splash effect 0.
The sprites are used as follows so you need to name all of them:
```cpp
// Animation of 8 frames for the 4 rotations
constexpr std::array<std::array<uint16_t, 8>, 4> SplashEffect1 = {
    { 29014, 29015, 29016, 29017, 29018, 29019, 29020, 29021 },
    { 29022, ...},
    ...
};
```
```
SPR_SPLASH_EFFECT_1_NORTH_0 = 29014,
SPR_SPLASH_EFFECT_1_NORTH_1
...
SPR_SPLASH_EFFECT_1_NORTH_7
SPR_SPLASH_EFFECT_1_EAST_0
...
```
Also you need to add the entries in the correct part of the sprites.h table not at the end.

**Comment by duncanspumpkin** [comment]
Also as this doesn't fully resolve #16033 avoid using Fix in the commit message just say. `Part of #16033`

**Comment by Gymnasiast** [comment]
@sammy-soto10 Could you address @duncanspumpkin ’s points?

**Comment by Gymnasiast** [comment]
As the points haven’t been addressed for six months, closing this.

**Pull request closed without merging**